Lib9c/Model/Garages/FungibleItemGarage.cs

```diff
@@ -0,0 +1,111 @@
+#nullable enable
+
+using System;
+using Bencodex.Types;
+using Nekoyume.Model.Item;
+
+namespace Nekoyume.Model.Garages
+{
+    public class FungibleItemGarage : IFungibleItemGarage
+    {
+        public IFungibleItem Item { get; }
+        public int Count { get; private set; }
+
+        public FungibleItemGarage(IFungibleItem item, int count)
+        {
+            Item = item ?? throw new ArgumentNullException(nameof(item));
+
+            if (count < 0)
+            {
+                throw new ArgumentOutOfRangeException(
+                    nameof(count),
+                    count,
+                    "Count must be greater than or equal to 0.");
+            }
+
+            Count = count;
+        }
+
+        public FungibleItemGarage(IValue? serialized)
+        {
+            if (serialized is null || serialized is Null)
+            {
+                throw new ArgumentNullException(nameof(serialized));
+            }
+
+            var list = (List)serialized;
+            Item = list[0].Kind == ValueKind.Null
+                ? throw new ArgumentNullException(nameof(serialized), "Item is null.")
+                : (IFungibleItem)ItemFactory.Deserialize((Dictionary)list[0]);
+            Count = (Integer)list[1];
+        }
+
+        public IValue Serialize() => Count == 0
+            ? (IValue)Null.Value
+            : new List(
+                Item?.Serialize() ?? Null.Value,
+                (Integer)Count);
+
+        public void Add(int count)
+        {
+            if (Count + count < 0)
```

Comment:
Is it possible for `count` to be negative? It seems a bit unusual to pass a negative value to the `Add` function, which is typically used for adding positive amounts.

Reply:
It was written simply to provide the addition operation in the class.
If you have a good name, please suggest it.
Or would it be better to change it to `SetCount(int value)` to do the summation outside?

Reply:
I simply thought form view of garage. On the garage's side, there are only `in` and `out` action makes change storage amount.
So If you don't have special meaning for the name `Add()` and `TransferTo()`, how about `Inbound()` and `Outbound()`? (those names are recommended by ChatGPT haha)

Reply:
Oh, that's nice. Or... `Load()` and `Unload()`? And maybe replace `TransferTo()` with `DeliverTo()`?
I'll think about it a bit more when I get back to the review! Thanks!

Reply:
I applied `Load()`, `Deliver()`, `Unload()` idea and you can figure out at #1959.